crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs

```diff
@@ -1,48 +1,99 @@
 use crate::{Diagnostic, DiagnosticId, Diagnostics};
-use bevy_app::prelude::*;
+use bevy_app::{prelude::*, PluginGroupBuilder};
 use bevy_core::FrameCount;
 use bevy_ecs::prelude::*;
 use bevy_time::Time;
 
-/// Adds "frame time" diagnostic to an App, specifically "frame time", "fps" and "frame count"
+/// This plugin group will add "frame time" diagnostics to an App, namely:
+/// * [`FrameTimeDiagnosticsPlugin`](crate::FrameTimeDiagnosticsPlugin)
+/// * [`FpsDiagnosticsPlugin`](crate::FpsDiagnosticsPlugin)
+/// * [`FrameCountDiagnosticsPlugin`](crate::FrameCountDiagnosticsPlugin)
+///
+#[derive(Default)]
+pub struct FrameTimeDiagnosticsPlugins;
```

Comment:
I like this direction, but agree on the need for a clearer name. `BasicPerformanceDiagnosticsPlugins` perhaps?